// react-native-country-picker-modal/src/CountryPicker.style.js
import { StyleSheet, PixelRatio } from 'react-native'
import { getHeightPercent } from './ratio'

export default StyleSheet.create({
  container: {},
  modalContainer: {
    
  },
  contentContainer: {
  },
  header: {
    flexDirection: 'row',
    alignItems: 'center',
    justifyContent: 'center',
    paddingLeft: getHeightPercent(4.0),
    paddingRight: getHeightPercent(2.2),
    paddingTop: getHeightPercent(1.0),
    paddingBottom: 0,
  },
  input: {
    height: 38,
    borderRadius: 12,
    textAlign: "center",
    flex: 1,
    backgroundColor: 'white',
    color: 'black'
  },
  inputOnly: {
    marginLeft: '15%'
  },
  touchFlag: {
    alignItems: 'center',
    justifyContent: 'center',
    height: getHeightPercent(1.9)
  },
  imgStyle: {
    resizeMode: 'contain',
    width: 25,
    height: 19,
    borderWidth: 1 / PixelRatio.get(),
    borderColor: '#eee',
    opacity: 0.8
  },
  emojiFlag: {
    alignItems: 'center',
    justifyContent: 'center',
    fontSize: 30,
    width: getHeightPercent(3.4),
    height: getHeightPercent(3.4),
    borderWidth: 1 / PixelRatio.get(),
    borderColor: 'transparent',
    backgroundColor: 'transparent'
  },
  itemCountry: {
    flexDirection: 'row',
    height: getHeightPercent(7),
    justifyContent: 'flex-start',
    alignItems: 'center',
    borderBottomWidth: 2 / PixelRatio.get(),
    borderBottomColor: '#ccc',
    paddingHorizontal: 0
  },
  itemCountryFlag: {
    justifyContent: 'center',
    alignItems: 'center',
    height: '7%',
    width: '15%'
  },
  itemCountryName: {
    justifyContent: 'space-between',
    flexDirection: 'row',
    marginTop: 4
  },
  countryName: {
    fontSize: getHeightPercent(2.2)
  },
  countryCode: {
    marginLeft: 4, 
    textAlign: 'left'
  },
  scrollView: {
    flex: 1
  },
  letters: {
    marginRight: 10,
    backgroundColor: 'transparent',
    justifyContent: 'center',
    alignItems: 'center'
  },
  letter: {
    height: 25,
    width: 20,
    justifyContent: 'center',
    alignItems: 'center'
  },
  letterText: {
    textAlign: 'center',
    fontSize: getHeightPercent(2.2)
  },
  closeButton: {
    height: 48,
    width: '15%',
    alignItems: 'center',
    justifyContent: 'center'
  },
  closeButtonImage: {
    height: 38,
    width: 38,
    resizeMode: 'contain'
  }
})
